logging/nox.py

```diff
@@ -36,7 +36,7 @@ def unit_tests(session, python_version):
     # Install all test dependencies, then install this package in-place.
     session.install(
         'mock', 'pytest', 'pytest-cov',
-        'flask', 'django', *LOCAL_DEPS)
+        'flask', 'webapp2', 'webob', 'django', *LOCAL_DEPS)
```

Comment:
They install fine in python3, you just can't use them.  I figure that's fine -- it lets us check that SyntaxError thing in the test -- so I'm happy enough to leave it like this.

But if you'd prefer to conditionally install the deps just for python2 -- under the theory nobody is going to be running python3 with webapp installed, so we could take out the SyntaxError handling -- that would make sense to me too.  I just don't know the syntax for it. :-)